### DISCUSSION TITLE
how to pass parent workflow node data to child workflow?

### DISCUSSION THREAD

**Discussion by 4kb-wgh** [elsa-workflows/elsa-core#1570] [Q&A]
Hi, @sfmskywalker .
This is my code in `Elsa 2.3`.
![image](https://user-images.githubusercontent.com/63579509/134304156-da35f8b4-8a9e-480f-800f-7f7cc8ac3a09.png)
![image](https://user-images.githubusercontent.com/63579509/134304428-a53ab1fa-fff7-4512-bf49-100c34cf3f8b.png)

**1. Is the way of parent workflow pass data to child workflow right?**
**2. How can i get data from parent workflow?**

**Comment by sfmskywalker** [comment]
Hi @4kb-wgh ,

Looks like you figured it out 😄 
Also be sure to check out [this guide](https://elsa-workflows.github.io/elsa-core/docs/next/guides/guides-parent-child-workflows) for a few examples related to parent/child workflows if you haven't already.

**Comment by zhangtingwz** [comment]
sorry ，i have a question ，i hope GetVariable from ActivityExecutionContext give my child workflow，how can i do ，please thank you very much
